nss-c31-mixed-havoc/nutshell
# Cm bootstrap

# Description

This merge request includes all of the functionality for the events section. It will contain bootstrap links on index.html. The event card should list both the current user's events and the user's friend's events. They will eventually have different styling, but currently the user's events has buttons and class myEvent. The friend's events have no buttons and class "friendEvent."

There was an addition of a new directory eventScripts/ with 5 javascript files inside that deal with the event section.

Applies to issue # 3

## Type of change

Please delete options that are not relevant.




# Testing Instructions

1. git fetch --all
2. git checkout cm-bootstrap
3. code .
4. cd src/lib
5. run grunt
6. login with user information
7. See that your events are listed with buttons, and friend events are not.
8. Check your database to make sure everyone has events/friends in order to be tested properly

# Checklist:
